content/docs/my-account-and-profile/sso.md

```diff
@@ -1,42 +1,34 @@
 ---
-lastUpdated: "01/25/2021"
+lastUpdated: "01/25/2022"
 title: "Single Sign-On (SSO)"
-description: "Learn how to enable SSO for your SparkPost account and get answers to common SSO questions."
+description: "Single Sign-On (SSO)"
 ---
 
-SparkPost supports single sign-on (SSO) through any SAML provider. SSO is enabled via the UI by an Admin level user. You can also set up [SCIM user provisioning through OKTA](https://www.sparkpost.com/docs/my-account-and-profile/scim/).
+Single sign-on (SSO) allows you login to your account using your company's preffered identify provider. You can enable SSO through any provider that supports SAML. You can also set up [SCIM user provisioning through OKTA](https://www.sparkpost.com/docs/my-account-and-profile/scim/).
 
 Check this [article](https://www.sparkpost.com/docs/faq/difference-between-starter-and-premier/) to verify that your account's plan level supports SSO.
 
-## Enable SSO
+## Enable Single Sign-On
 
 Enabling SSO is a 3-step process:
 
 ### 1. Enable for Account:
-Go to your [Account Settings](https://app.sparkpost.com/account/settings)([EU](https://app.eu.sparkpost.com/account/settings)) and click on "Provision SSO" to upload your SAML configuration file. Once provisioned, you can enable SSO for the account.
-
-### 2. Enable for users:
-Update your users in from the [Users](https://app.sparkpost.com/account/users)([EU](https://app.eu.sparkpost.com/account/users)) page to enable SSO.
-
-### 3. Configure your identity provider with the callback URL:
-1. Your identity provider will need the callback URL for SparkPost. Please refer to your identity provider for specific instructions on where to submit this information.
-1. The callback URL is:
-US: `https://api.sparkpost.com/api/v1/users/saml/consume`
-
-EU: `https://api.eu.sparkpost.com/api/v1/users/saml/consume`
+Go to your [Account Settings page](https://app.sparkpost.com/account/settings)([EU](https://app.eu.sparkpost.com/account/settings)) and click on "Provision SSO" to upload your SAML configuration file. Copy the callback URL to configure your identity provider.
```

Comment:
Should we include some screenshots on this page as well? Again, I feel like this would be a good addition for a better UX.